XML3-Homework/parrish_1-26_xml3.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<root>
+    <intro> Learning to knit can seem a bit daunting, so if you're keen to kick start your knitting
+        journey but you're not sure where to begin, you’ve come to the right place! We've got loads
+        of brilliant learn to knit videos that cover everything from beginner tutorials on how to
+        hold the needles and knit your first row, through to more advanced knitting techniques such
+        as how to work those tricky button holes. You'll be knitting up a storm in no time! </intro>
+
+    <extraInfo> Getting started Before you begin your journey of learning how to knit, you’ll need a
+        few essentials to get you started. We'll talk you through the essential first steps to
+        getting started including picking a knitting project, choosing the best needles for the job
+        and selecting your yarn, as well as showcasing some fabulous kits. Then enjoy browsing our
+        gallery of step-by-step tutorials. </extraInfo>
+
+    <mainSteps>
+        <step main="first">1. Pick a pattern</step>
+        <extraInfo> The first step on your knitting odyssey is to pick a pattern! We've got hundreds
+            of knitting projects from a host of designers and much-loved brands for all abilities.
+            Whether you're looking to craft a cosy hat for a buddy, a snuggly baby blanket, or
+            dreamy scarf, we have a beautiful pool of projects to inspire you! </extraInfo>
+        <step main="second">2. Choose the best needles to get started</step> First up, if you’re
+        learning traditional knitting techniques, we would recommend starting with single point
+        needles, which are used in our beginner videos. But there’s a whole menagerie of knitting
+        needles to explore from DPNs to circular in different sizes and materials, from bamboo to
+        plastic. <step main="third">3. Select your yarn</step> Yarn glorious yarn! One of the
+        greatest joys a knitter has on her way to making divine projects is choosing which yummy
+        yarn to work with. With so many yarns to choose from in every shade imaginable, it can
+        easily get a knitting newbie in a tangle. From selecting a suitable fiber to exploring
+        color, we’re here to help! <step main="fourth">4. Cast on</step> Before you start knitting
+        the very first step to knitting is learning how to tie a slip knot and cast on the number of
+        stitches you will be knitting in your row. See our step-by-step guide and video on how to
+        tie a slip knot or follow these simple steps! </mainSteps>
+
+    <slipKnot> How to tie a slip knot and cast on <slipKnotSteps slipKnot="first"> 1 Leaving a nice
+            long tail (which you’ll sew in later), take the yarn and wind it around your fingers. </slipKnotSteps>
```

Comment:
@kzp308 I like how you're thinking about the structure of your code here in this `<slipKnot>` element: You're basically using your elements now to section off related info. Good.